test/integration/aaa_download_only_test.go

```diff
@@ -54,26 +53,31 @@ func TestDownloadOnly(t *testing.T) {
 		constants.NewestKubernetesVersion,
 	}
 
+	// Small optimization, don't run the exact same set of tests twice
+	if constants.DefaultKubernetesVersion == constants.NewestKubernetesVersion {
+		versions = versions[:len(versions)-1]
+	}
+
 	for _, v := range versions {
 		t.Run(v, func(t *testing.T) {
 			defer PostMortemLogs(t, profile)
 
-			// --force to avoid uid check
-			args := append([]string{"start", "-o=json", "--download-only", "-p", profile, "--force", "--alsologtostderr", fmt.Sprintf("--kubernetes-version=%s", v)}, StartArgs()...)
-
-			rt, err := Run(t, exec.CommandContext(ctx, Target(), args...))
-			if rrr == nil {
-				// Preserve the initial run-result for debugging
-				rrr = rt
-			}
-			if err != nil {
-				t.Errorf("failed to download only. args: %q %v", args, err)
-			}
 			t.Run("check json events", func(t *testing.T) {
```

Comment:
how about choosing shorter names ? so it shows better in gopogh url ?
"json-events" instead "check json events"

Reply:
and same for other names

Reply:
done